### PR TITLE
Add scrollbar widget

### PR DESCRIPTION
**Pull request by Correct-Syntax**
This is WIP and currently just the code from https://github.com/sheyballs/custom-wxpy/blob/main/SuperScrolledWindow.py (which I have permission to use from the author).

Things to do:

- [ ] Fix the jumpiness when you first click on the scrollbars
- [ ] Edit the color scheme and visual elements to fit the Gimel Studio style
- [ ] Add demonstration to ``demo.py``

### CODE REVIEW

**Comment by Correct-Syntax** [comment]
Note: I am not sure why it is wanting to re-add the previous commits again!...

File: gswidgetkit/scrollbar.py [hunk 1]
```diff
@@ -13,3 +13,431 @@
...
+            self.panel.Layout()
+
+    app = wx.App(False)
+    frame = TestFrame()
+
+    frame.Show()
+
+    app.MainLoop()
+    app.ExitMainLoop()
+    app.Destroy()
```
Comment (iwoithe) [inline gswidgetkit/scrollbar.py:443]: Move the demo to ``demo.py`` (so it is consistent with all widgets in this repository)

**Review by iwoithe — COMMENTED** [review]
``scrollbar.py`` looks fine so far (although I haven't been able to test it myself). I will look more in depth into the code when you are ready to merge it. :slightly_smiling_face:

> Note: I am not sure why it is wanting to re-add the previous commits again!...

I believe it is because you are doing all changes on your "master" branch. Ideally, a new branch should be used for each pull request.

File: gswidgetkit/scrollbar.py (hunk 1 above)
Comment (iwoithe) [inline gswidgetkit/scrollbar.py:443]: Move the scrollbar demo to ``demo.py``

Comment (Correct-Syntax) [reply gswidgetkit/scrollbar.py:443]: Agreed. This would be resolved via "Add demonstration to demo.py" 😃

**Comment by Correct-Syntax** [comment]
> I believe it is because you are doing all changes on your "master" branch. Ideally, a new branch should be used for each pull request.

That is true. Though maybe its because I did a rebase before merging. 😅

**Comment by Correct-Syntax** [comment]
I think I will close this and restart over as this has conflicts since I pushed to ``master`` directly.

**Pull request closed without merging**